src/brogue/Items.c

```diff
@@ -2483,10 +2484,8 @@ void itemDetails(char *buf, item *theItem) {
                         (theItem->charges == 1 ? "" : "s"));
                 strcat(buf, buf2);
 
-                if ((theItem->charges < RING_DELAY_TO_AUTO_ID || (theItem->flags & (ITEM_MAGIC_DETECTED | ITEM_IDENTIFIED)))
-                    && theItem->enchant1 > 0) { // Mention the unknown-positive-ring footnote only if it's good magic and you know it.
-
-                    sprintf(buf2, ", and until you understand its secrets, it will function as a +%i ring.", theItem->timesEnchanted + 1);
+                if ((theItem->charges < RING_DELAY_TO_AUTO_ID || (theItem->flags & (ITEM_MAGIC_DETECTED | ITEM_IDENTIFIED)))) {
```

Comment:
What do you think the purpose of the condition `(theItem->flags & (ITEM_MAGIC_DETECTED | ITEM_IDENTIFIED))` is? This is in the else branch of an `ITEM_IDENTIFIED` conditional, so that flag at least seems redundant. And then, if a ring is magic-detected, but not identified, then surely `theItem->charges < RING_DELAY_TO_AUTO_ID` will still be true, making that one redundant too?